scripts/qesap/test/unit/test_qesap_lib_cmd_ansible.py

```diff
@@ -0,0 +1,47 @@
+from unittest import mock
+import os
+import logging
+import yaml
+
+from lib.cmds import cmd_ansible
+
+log = logging.getLogger(__name__)
+
+
+@mock.patch("lib.process_manager.subprocess_run")
+def test_cmd_ansible(subprocess_run, tmpdir):
+    '''
+    This test checks if the ansible execution generates log files
```

Comment:
Which part of the test `checks if the ansible execution generates log files` ?

Reply:
refactored

Reply:
```suggestion
   Given an ansible-playbook command line and some string to simulate an ansible stdout, this test verify that the function write them to file.
```